<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/channeltoolbarwidget.ui -->
<!-- Phiên bản: 1.1 (Thêm các nút Thêm/Sửa/Xóa/Nhập) -->
<ui version="4.0">
 <class>ChannelToolbarWidget</class>
 <widget class="QWidget" name="ChannelToolbarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="toolbarFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="addToolButton">
        <property name="text">
         <string>+ Thêm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="importToolButton">
        <property name="text">
         <string>Nhập từ File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editToolButton">
        <property name="text">
         <string>Sửa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteToolButton">
        <property name="text">
         <string>Xóa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="upToolButton">
        <property name="text">
         <string>↑ Lên</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="downToolButton">
        <property name="text">
         <string>↓ Xuống</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="settingsToolButton">
        <property name="text">
         <string>⚙️</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

